src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -899,6 +908,21 @@ class PublishArgument(cli.Argument):
             "required": False,
         },
     )
+    MAX_BUILD_FIND_WAIT = cli.ArgumentProperties(
```

Comment:
Any particular reason we use `MAX_..._WAIT` instead of `..._TIMEOUT`, which feels more accurate in this context

Reply:
To emphasize the resemblance with [`--max-build-processing-wait`](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/publish.md#--max-build-processing-wait--wmax_build_processing_wait), nothing more really. As for all copy related things I'm looking for input from @helinanever.